.goreleaser.yml

```diff
@@ -0,0 +1,28 @@
+# https://github.com/crazy-max/goreleaser-xx
+
+env:
+  - GO111MODULE=auto
+
+builds:
+  - mod_timestamp: '{{ .CommitTimestamp }}'
+    flags:
+      - -trimpath
+
+snapshot:
+  name_template: "{{ .Tag }}-next"
+
+brews:
+  - goarm: "7"
+    tap:
+      owner: jzelinskie
+      name: homebrew-faq
+    folder: Formula
+    homepage: https://github.com/jzelinskie/faq
+    description: Format Agnostic jQ
+    license: "Apache 2.0"
+    dependencies:
+      - name: jq
+    test: |
+      system "#{bin}/faq", "--version"
+    # FIXME: Skip upload for now. Needs to handle multiple packages.
+    skip_upload: true
```

Comment:
I have started to add support for homebrew taps but it upload is skipped for now. We need to handle multiple packages outside the GoReleaser pipeline or maybe we could use an internal invocation. We can do that in a follow-up if that's ok for you.